examples/stm32/xively_demo_stm32f4xx_nucleo_eth/SW4STM32/syscalls.c

```diff
@@ -133,14 +133,9 @@ int _lseek(int file, int ptr, int dir)
 
 int _read(int file, char *ptr, int len)
 {
-	int DataIdx;
-
-	for (DataIdx = 0; DataIdx < len; DataIdx++)
-	{
-	  *ptr++ = __io_getchar();
-	}
-
-   return len;
+    /* scanf calls _read() with len=1024, so eat one character at time */
+    *ptr = __io_getchar();

```

Comment:
would be good to figure out why scanf calls _read with 1024 for ethernet version and probably not? for wifi version. Or does wifi version scanf behave same?